app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -108,11 +112,9 @@ const AvailableActions = ({ course, current_user, updateCourse, courseCreationNo
       controls.push((<div key="search" className="available-action"><a href={`/tickets/dashboard?search_by_course=${course.slug}`} className="button">{I18n.t('courses.search_all_tickets_for_this_course')}</a></div>));
     }
     // If course is not published, show the 'delete' button to instructors and admins.
-    // Show a disabled version of it on P&E Dashboard even if a course is published,
-    // so that users can see the instructions for how to enable deletion.
     if ((user.isAdvancedRole || user.admin) && (!course.published || !Features.wikiEd)) {
       controls.push((
-        <div title={I18n.t('courses.delete_course_instructions')} key="delete" className="available-action">
+        <div title={Features.wikiEd ? I18n.t('courses.delete_course_instructions') : undefined} key="delete" className="available-action">
```

Comment:
What's this change for?

Reply:
From my understanding the old if statement translates as: 

Assuming the user has an advanced role or is an admin:

1. **If the course is not published:**
-For both WikiEd and (P&E), the title attribute will be set to the instructions, which display on hover.
2. **If the course is published:**
-In WikiEd mode, the button will not be displayed at all, because the condition fails due to !Features.wikiEd being false.
-In P&E mode, the title attribute will be set to the instructions, which display on hover.

Since the desired behaviour is to stop the "You may delete this page if all campaigns are removed first. " instructions from showing in P&E, the added condition only sets the title to the instructions if in WikiEd mode.

So basically, it only displays on hover if the course is not published and in WikiEd mode.